scripts/bootloader/generate_zip.py

```diff
@@ -52,22 +52,26 @@ def get_name(text_wrapper):
         manifest['name'] = args.name
 
     if args.meta_info_file is not None:
+        meta = None
         meta_file = PurePath(args.meta_info_file)
         if Path(meta_file).is_file():
             with Path(meta_file).open('r') as f:
                 meta = yaml.safe_load(f.read())
+        else:
+            print("WARNING: {} not found".format(args.meta_info_file))
```

Comment:
I understand the reason for doing this, but isn't it better with a hard failure in case this script is called with a non-existing meta file ?

A hard failure will make it faster to discover an error than a warning which may go unnoticed.

Reply:
Good point.  I originally added it as a debugging aid.  I just changed it to raise an OSError instead.